src/coreclr/jit/importercalls.cpp

```diff
@@ -6301,11 +6295,12 @@ void Compiler::impMarkInlineCandidate(GenTree*               callNode,
         assert(call->GetInlineCandidatesCount() > 0);
         for (uint8_t candidateId = 0; candidateId < call->GetInlineCandidatesCount(); candidateId++)
         {
-            InlineResult inlineResult(this, call, nullptr, "impMarkInlineCandidate", true);
+            InlineResult inlineResult(this, call, nullptr, "impMarkInlineCandidate for GDV");
 
             // Do the actual evaluation
             impMarkInlineCandidateHelper(call, candidateId, exactContextHnd, exactContextNeedsRuntimeLookup, callInfo,
                                          ilOffset, &inlineResult);
+            inlineResult.Report();
```

Comment:
Calls Report explicitly instead of relyin on destructor

Reply:
What went wrong with relying on the destructor?

Reply:
Reverted to destructor since it raised a question - I don't have a strong opinion about it